subsys/bluetooth/audio/bap_broadcast_assistant.c

```diff
@@ -453,12 +453,12 @@ static void long_bap_read(struct bt_conn *conn, uint16_t handle)
 		if (err != 0) {
 			LOG_DBG("Failed to get conn info, use default interval");
 
-			conn_info.le.interval = BT_GAP_INIT_CONN_INT_MIN;
+			bt_conn_le_set_interval_1250_us_units(&conn_info, BT_GAP_INIT_CONN_INT_MIN);
```

Comment:
I don't want this function. Bootleging application data in `struct bt_conn_info` is not reasonably part of the API, and we don't need to support it.